src/main/java/com/direwolf20/buildinggadgets/common/entities/BlockBuildEntity.java

```diff
@@ -21,14 +21,17 @@
 import java.util.Optional;
 
 public class BlockBuildEntity extends EntityBase {
-    private static final DataParameter<Integer> toolMode = EntityDataManager.<Integer>createKey(BlockBuildEntity.class, DataSerializers.VARINT);
+
+    public static final int MODE_PLACE = 1;
+    public static final int MODE_REMOVE = 2;
+    public static final int MODE_REPLACE = 3;
```

Comment:
While you're at it, could these be enum constants?

Reply:
I would agree with you if we don't need to convert the enums during serialization/deserialization, and for `DataParameter<T>` in some way. Either `ordinal()` or `name()`.

Reply:
For serilisation/deserilisation  it would be easy to. Assign an ID...